### PR TITLE
refactor(w3c/headers): use htmlJoinAnd for wg[Patent]HTML

### PR DESCRIPTION
**Pull request by SuyashSalampuria**
Fixes partially #2551 
refactor: convert [conf.bugTrackerHTML] to conf.bugTrackerHTML ,[conf.bugTrackerHTML] to conf.bugTrackerHTML and used htmlJoinAnd at one more place in core/headers.js

### CODE REVIEW

**Comment by SuyashSalampuria** [comment]
@saschanaz  Please review this PR

**Comment by saschanaz** [comment]
It seems we don't have any tests that covers `conf.bugTracker`. Filed #2632 for this, and I think we should merge this after fixing it.

**Comment by marcoscaceres** [comment]
@SuyashSalampuria make sure you write clear pull request titles (i.e., not everything you changed there:)), then write what you change as comment.

**Comment by SuyashSalampuria** [comment]
@saschanaz  I think it goes fine now. Please review it once

File: src/w3c/templates/sotd.js [hunk 1]
```diff
@@ -330,7 +330,7 @@ function linkToWorkingGroup(conf) {
   }
   return html`
     <p>
-      This document was published by ${[conf.wgHTML]} as ${conf.anOrA}
+      This document was published by ${conf.wgHTML} as ${conf.anOrA}
```
Comment (saschanaz) [inline src/w3c/templates/sotd.js]: Could you do the same for `conf.wgPatentHTML`? Thanks!

Comment (SuyashSalampuria) [reply src/w3c/templates/sotd.js]: > Could you do the same for `conf.wgPatentHTML`? Thanks!
I think that it has been done already. please correct if you it somewhere

Comment (saschanaz) [reply src/w3c/templates/sotd.js]: Ah, this time it's just `${[wgPatentHTML]}` without conf 😁

Comment (saschanaz) [reply src/w3c/templates/sotd.js]: And also while you are at it could you also do it for `wgPatentURI`?

Comment (SuyashSalampuria) [reply src/w3c/templates/sotd.js]: Sure, I did that already. just about to make the PR

**Comment by SuyashSalampuria** [comment]
> LGTM +1, will file a new issue for the URL thing.

Shall I try filing the issue for it? @saschanaz :)

**Comment by saschanaz** [comment]
Ah, actually I was just to wait for the answer to https://github.com/w3c/respec/pull/2636#issuecomment-565788167. (I thought I wrote about it here 😅)

**Comment by saschanaz** [comment]
>Shall I try filing the issue for it?

But.... Please do, as it seems a good idea 👍

**Pull request merged**